src/extension.ts

```diff
@@ -54,15 +54,31 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(vscode.commands.registerCommand(stopBuildCommandId, stopBuild));
     context.subscriptions.push(vscode.commands.registerCommand(compileThisFileCommandId, compileThisFile));
 
+    context.subscriptions.push(vscode.commands.registerCommand(
+        launchTargetPathId,
+        async () => {
+            if (currentTarget === null) {
+                return null;
+            }
+            // TODO: I'm not sure if this is the best way to ensure that
+            // when we're getting the path to the build target for
+            // debugging or launching, we build the latest version of
+            // the underlying binary.
+            await build();
```

Comment:
You can run `buck2 targets $TARGET_NAME --output-path` to get the path where an executable will go without paying to actually do the build.

Reply:
I do that in the line below, this is so that when you're debugging in VSCode you never need to remember to rebuild the binary, you just start debugging and the latest binary is there for you.